src/Http/Controllers/Dominion/TownCrierController.php

```diff
@@ -15,9 +15,6 @@ public function getIndex()
 
         $realm = $dominion->realm;
         $realmieDominionIds = $realm->dominions
```

Comment:
Can you change this variable to $dominionIds?

Reply:
Sure

Reply:
Also in the compact() call below, and in the view

Reply:
Solved